### PR TITLE
MsgBoxes for Crowdloans and Auction pages

### LINKED ISSUE #5854: Parachain Slot Acquisition note about coretime superseding them
Just add a Warning or Note about auctions being soon deprecated in this section -> https://wiki.polkadot.network/docs/learn-parachains#parachain-slot-acquisition

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-auction.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ keywords: [auction, slot auctions, parachain, bidding]
 slug: ../learn-auction
 ---
 
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
 import RPC from "./../../components/RPC-Connection";
 
+<MessageBox message="Parachain Slot Auctions are being soon deprecated. For existing parachains, leases will be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```
Comment (profradha) [inline docs/learn/learn-auction.md]:
```suggestion
<MessageBox message="Parachain Slot Auctions will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. For existing parachains, the remainder of the lease will automatically be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```

File: docs/learn/learn-crowdloans.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ keywords: [crowdloans, parachains, lending, auction]
 slug: ../learn-crowdloans
 ---
 
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
 import RPC from "./../../components/RPC-Connection";
 
+<MessageBox message="With Agile Coretime, crowdloans are being soon deprecated. For an alternative option to crowdloans see the [Polimec parachain](https://www.polimec.org/)." />
```
Comment (profradha) [inline docs/learn/learn-crowdloans.md]:
```suggestion
<MessageBox message="Crowdloans will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. As an alternative for fundraising in a decentralized, transparent, and regulatory compliant manner within the ecosystem, check out the [Polimec parachain](https://www.polimec.org/)." />
```

File: docs/learn/learn-parachains-faq.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ keywords: [parachains, application-specific, sharding, faq]
 slug: ../learn-parachains-faq
 ---
 
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
+
+<MessageBox message="Parachain Slot Auctions and Crowdloans are being soon deprecated. For existing parachains, leases will be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```
Comment (profradha) [inline docs/learn/learn-parachains-faq.md]:
```suggestion
<MessageBox message="Parachain Slot Auctions and Crowdloans will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. For existing parachains, the remainder of the lease will automatically be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```

File: docs/learn/learn-parachains.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ keywords: [parachains, application-specific, sharding]
 slug: ../learn-parachains
 ---
 
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
+
+<MessageBox message="Parachain Slot Auctions are being soon deprecated. For existing parachains, leases will be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```
Comment (profradha) [inline docs/learn/learn-parachains.md]:
```suggestion
<MessageBox message="Parachain Slot Auctions will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. For existing parachains, the remainder of the lease will automatically be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
```

**Review by DrW3RK — APPROVED** [review]
Timely update, thank you! Let's revisit these pages after coretime is activated on Polkadot.

**Pull request merged**